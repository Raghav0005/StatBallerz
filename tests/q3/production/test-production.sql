DELETE FROM Users WHERE username = 'user50';