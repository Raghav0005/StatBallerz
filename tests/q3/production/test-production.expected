DB20000I  The SQL command completed successfully.

